docs/tracking/google-tag-manager.md

```diff
@@ -29,11 +34,19 @@ const sdk = createInstance({
   ) {
     // push to dataLayer here
     window.dataLayer.push({
-      event: 'featurevisor-activation',
```

Comment:
for the sake of having the name as one word, we can write it as `featurevisorActivation`.

Reply:
Yes @fahad19 I agree with the change here, best practice for the event pushes is camelcase and I wasn’t sure if you preferred it in one word or not :)